pkgs/development/tools/packer/default.nix

```diff
@@ -40,7 +44,178 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     mkdir -p $out/bin
-    mv bin/* $out/bin
+    mv bin/* $out/bin # */
+  '';
+
+  passthru.updateScript = writeScript "update-packer" ''
+    #!${bash}/bin/bash
```

Comment:
ok, it's something inconsistent in packer vendor dir, it looks like it's half managed by `govendor` without all deps checked in into its repo